### ISSUE TITLE
MD011: false positive for parenthesis plus footnote

### ISSUE THREAD

**Issue by ndarville** [DavidAnson/markdownlint#18]
I first posted this to the Ruby project by mistake, but at least means that both Ruby and Node users will benefit from it.

Here is what it originally read:

* * * *

```markdown
(...)[^footnote]
```

Is regarded as a reverse link, because markdownlint considers footnote syntax (part Markdown Extra) a link.

* * * *

<https://github.com/mivok/markdownlint/issues/132>

**Comment by DavidAnson** [comment]
markdownlint supports the CommonMark specification, not Markdown Extra, so there are probably other incompatibilities. That said, this particular one seems fairly easy to special-case, so I'll do so in a future release. Thanks for reporting it!
